pkg/agent/openflow/client.go

```diff
@@ -174,13 +173,9 @@ func (c *client) InstallNodeFlows(hostname string,
 ) error {
 	c.replayMutex.RLock()
 	defer c.replayMutex.RUnlock()
-	flows := make([]binding.Flow, 2, 3)
-	flows[0] = c.arpResponderFlow(peerGatewayIP, cookie.Node)
-	flows[1] = c.l3FwdFlowToRemote(localGatewayMAC, peerPodCIDR, tunnelPeerAddr, tunOFPort, cookie.Node)
-	if tunnelPeerAddr == nil {
-		// Not the default (flow based) tunnel. Add a separate tunnelClassifierFlow.
-		flows = append(flows, c.tunnelClassifierFlow(tunOFPort, cookie.Node))
-	}
+	flows := []binding.Flow{
+		c.arpResponderFlow(peerGatewayIP, cookie.Node),
+		c.l3FwdFlowToRemote(localGatewayMAC, peerPodCIDR, tunnelPeerAddr, tunOFPort, cookie.Node)}
```

Comment:
do you want to keep `tunOFPort` as a parameter or revert that change (given that it is always called with the default tunnel port now)?

Reply:
I feel still cleaner to let the caller provides the OFPort, instead of making ofclient know about the defaultTunnelOFPort.

Reply:
Sounds good.